nym-vpn-cli/src/main.rs

```diff
@@ -123,13 +123,6 @@ async fn run_vpn(args: commands::RunArgs, data_path: Option<PathBuf>) -> Result<
             .map(|url| url.to_string())
             .unwrap_or("unavailable".to_string())
     );
-    info!(
```

Comment:
Keep this?

Reply:
I didn't intend to remove it. Yes, I will keep it